frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -1,14 +1,15 @@
+local ConfirmBox = require("ui/widget/confirmbox")
 local DataStorage = require("datastorage")
 local Device = require("device")
 local DictQuickLookup = require("ui/widget/dictquicklookup")
-local InputContainer = require("ui/widget/container/inputcontainer")
 local InfoMessage = require("ui/widget/infomessage")
+local InputContainer = require("ui/widget/container/inputcontainer")
 local JSON = require("json")
+local Screen = require("device").screen
```

Comment:
Yeah, that's basically what we've already been doing. Sticking it in like this doesn't help much against the problem I mentioned in any case.